components/brave_search/browser/brave_search_fallback.cc

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_search/browser/brave_search_fallback.h"
+
+#include <utility>
+
+namespace brave_search {
+
+BraveSearchFallback::BraveSearchFallback() : weak_factory_(this) {}
+
+BraveSearchFallback::~BraveSearchFallback() {}
+
+void BraveSearchFallback::FetchBackupResults(
+    const std::string& query_string,
+    const std::string& lang,
+    const std::string& country,
+    const std::string& geo,
+    FetchBackupResultsCallback callback) {
+  // TODO(sergz): make a call to fetch a response and remove that callback call
+  std::move(callback).Run("response");
```

Comment:
we probably need some error handling in this callback. An `int32` to pass a value from net_error_list.h? Then default to `Run("", NET_ERROR_ABORTED)` for now?

Reply:
actually can you call `OnFetchBackupResults(callback, "", NET_ERROR_ABORTED)`? I was a little confused about `OnFetchBackupResults` at first

Reply:
that is an upcoming task and it will be done in it's context when it's actually connected, yeah there are perhaps extra arguments we want to pass